radio/src/hal/module_port.h

```diff
@@ -145,10 +145,16 @@ inline uint8_t modulePortGetType(const etx_module_driver_t& d) {
 }
 
 inline const etx_serial_driver_t* modulePortGetSerialDrv(const etx_module_driver_t& d) {
+#if defined(SIMU)
+  if (d.port == nullptr) return nullptr;
+#endif
   return d.port->drv.serial;
 }
 
 inline const etx_timer_driver_t* modulePortGetTimerDrv(const etx_module_driver_t& d) {
+#if defined(SIMU)
+  if (d.port == nullptr) return nullptr;
+#endif
```

Comment:
Here as well.

Reply:
This is changed by @philmoz instead, any comments.

Reply:
I added these checks to stop the simulator from crashing. I suspect it's because the simulator does not have a real module and the initialisation is not the same a on a real radio; but I did not dig very deeply into it.

Reply:
Sure, but we could still add these checks whether we're on SIMU or not, right? Do you guys see any downside?

Reply:
I will change it anyway, check always there.

Reply:
@raphaelcoeffic Changed as per your request.